db/schema.rb

```diff
@@ -78,6 +78,7 @@
     t.integer "defence_statement"
     t.integer "number_of_witnesses"
     t.string "supplemental_claim"
+    t.string "preparation_time"
```

Comment:
**nit** seems to be a result of local db differences and could be excluded from diff